### PR TITLE
[LPT] Introduced BiasAttribute

### PR DESCRIPTION
**Pull request by v-Golubev**
### Details:
This PR adapts LPT behavior to the callback for AddTransformation which should be set in CPU in order to disable DQ operations propagation through biases.

### Tickets:
 - *106503*

### CODE REVIEW

**Review by eshoguli — APPROVED** [review]
comments will be fixed in separate PR

File: src/plugins/intel_cpu/tests/functional/subgraph_tests/src/fq_layer_dq_bias.cpp [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        targetDevice = CommonTestUtils::DEVICE_CPU;
+        fusedOps = std::vector<std::string>{"Add"};
+        std::tie(inFmts, outFmts, priority, selectedType) = CPUSpecificParams{{}, {}, {}, CPUTestsBase::any_type};
+        std::unordered_map<std::string, std::string> ngraph_type_to_plugin_type{
+            {"Convolution", "Convolution"},
+            {"GroupConvolution", "Convolution"},
+            {"ConvolutionBackpropData", "Deconvolution"},
+            {"MatMul", "MatMul"},
+            {"MatMulWithConstant", "FullyConnected"},
+        };
```
Comment (maxnick) [inline src/plugins/intel_cpu/tests/functional/subgraph_tests/src/fq_layer_dq_bias.cpp:54]: Is not it better to use `static const std::unordered_map<std::string, std::string> ngraph_type_to_plugin_type`?

**Review by maxnick — APPROVED** [review]
CPU plugin part LGTM. There is one minor comment left.

File: src/common/low_precision_transformations/src/eltwise_base_transformation.cpp [hunk 2]
```diff
@@ -70,8 +71,18 @@ static std::shared_ptr<Node> getDataParent(const std::shared_ptr<Node> branchDat
         parent = parent->get_input_node_shared_ptr(0);
     }
 
-    if (ov::is_type<opset1::Add>(parent) && isTargetType(parent->get_input_node_shared_ptr(0))) {
-        return parent->get_input_node_shared_ptr(0);
+    if (ov::marked_as_bias(parent)) {
+        const auto bias_parent = parent->get_input_node_shared_ptr(0);
+        // target node just before bias
+        if (isTargetType(bias_parent)) {
```
[the remark this answers is not in the document]
Comment (v-Golubev) [reply src/common/low_precision_transformations/src/eltwise_base_transformation.cpp:77]: If we extend this list, branch selection algorithm might negatively affect performance on some models (all details can be shared offline). So I suggest leave the current logic.

File: src/common/low_precision_transformations/tests/simple_low_precision_transformer.cpp [hunk 3]
```diff
@@ -26,6 +27,7 @@ SimpleLowPrecisionTransformer::SimpleLowPrecisionTransformer(
 
     // TODO: use one pass manager
     markup = std::make_shared<ngraph::pass::Manager>(passConfig);
+    markup->register_pass<ngraph::pass::low_precision::MarkupBias>();
```
[the remark this answers is not in the document]
Comment (v-Golubev) [reply src/common/low_precision_transformations/tests/simple_low_precision_transformer.cpp:30]: Applied in #16856

File: src/plugins/intel_cpu/tests/functional/subgraph_tests/src/fq_layer_dq_bias.cpp (hunk 1 above)
Comment (v-Golubev) [reply src/plugins/intel_cpu/tests/functional/subgraph_tests/src/fq_layer_dq_bias.cpp:54]: Applied in #16856
Also a short description added for the test class.

**Pull request merged**